common/lib/capa/capa/inputtypes.py

```diff
@@ -818,8 +818,15 @@ def setup(self):
         self.setup_code_response_rendering()
 
     def _extra_context(self):
-        """Defined queue_len, add it """
-        return {'queue_len': self.queue_len, }
+        """Define queue_len, arial_label and code mirror exit message context variables"""
+        _ = self.capa_system.i18n.ugettext
+        return {
+            'queue_len': self.queue_len,
+            'aria_label': _("{programming_language} editor").format(
```

Comment:
NIT: use single quotes since we're already using them at most of the places in this method.